lib/webrick/httpservlet/filehandler.rb

```diff
@@ -529,7 +529,7 @@ def set_dir_list(req, res)
             dname = name
           end
           s =  +"<TR><TD class=\"name\"><A HREF=\"#{HTTPUtils::escape(name)}#{query if name.end_with?('/')}\">#{HTMLUtils::escape(dname)}</A></TD>"
-          s << "<TD class=\"mtime\">" << (time ? time.strftime("%Y/%m/%d %H:%M") : "") << "</TD>"
+          s << "<TD class=\"mtime\">" << (time ? time.strftime("%Y-%m-%d %H:%M") : "") << "</TD>"
```

Comment:
I think there is two difficulties

1. Maybe there already exist log analyzers uses the current format even if `%Y-%m-%d` is better to read for people in some locales.
2. ISO 8601 needs putting `T` between `%Y-%m-%d` and  `%H:%M`, i.e., `%Y-%m-%dT%H:%M`. And it's not clear which is better, `"%Y/%m/%d %H:%M` or `%Y-%m-%dT%H:%M`.